cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -197,34 +230,18 @@ func (dp *DevicePlugin) getDpdkMounts(dpdkDeviceName string) []pluginapi.Mount {
 	}
 }
 
-func (dp *DevicePlugin) getDeviceID(pciAddr string) (string, error) {
-	devID, err := os.ReadFile(filepath.Join(dp.pciDeviceDir, filepath.Clean(pciAddr), "device"))
+func getDeviceID(device string) (string, error) {
+	devID, err := os.ReadFile(filepath.Join(device, "device"))
 	if err != nil {
-		return "", errors.Wrapf(err, "Cannot obtain ID for the device %s", pciAddr)
+		return "", err
```

Comment:
Errors returned by external libs don't contain a trace stack, better have it wrapped.

Reply:
I left it out here because it's just printed out as a warning.

Reply:
That's fine. I was trying to adhere the policy we documented in DEVEL.md.
Maybe a comment would be helpful here then.

Reply:
I decided to wrap to make it consistent with everything else.